invenio_app_ils/ui/src/invenio_app_ils/common/config/invenioConfig.js

```diff
@@ -1,6 +1,9 @@
 import _get from 'lodash/get';
 
 export const invenioConfig = {
+  authors: {
```

Comment:
I would have used something more specific maybe here... e.g. `documentEditorAuthors`... WDYT? Maybe for the next PR?

Reply:
I named the key `authors` since we probably will need more configuration for them in the future. E.g. we might need to configure the number of authors to display on the frontsite, backoffice, for items, etc. But `maxDisplay` definitely needs to be more specific.